swagger.json

```diff
@@ -2574,9 +2574,234 @@
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            },
+            "delete": {
+                "tags": [
+                    "Default", "Fleet"
+                ],
+                "summary": "/addresses/{addressId}",
+                "description": "Delete an address",
+                "operationId": "DeleteOrganizationAddress",
+                "parameters": [
+                    { "$ref": "#/parameters/accessTokenParam" },
+                    {
+                        "name": "addressId",
+                        "in": "path",
+                        "required": true,
+                        "description": "ID of the address/geofence",
+                        "type": "integer",
+                        "format": "int64"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Address was successfully deleted. No response body is returned."
+                    },
+                    "default": {
+                        "description": "Unexpected error.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            }
         }
     },
     "definitions": {
+        "Address": {
+            "type": "object",
+            "description": "Information about an address/geofence. Geofences are either a circle or a polygon.",
+            "properties": {
+                "id": {
+                    "type": "integer",
+                    "description": "ID of the address",
+                    "format": "int64",
+                    "example": 123
+                },
+                "name": {
+                    "type": "string",
+                    "description": "Name of the address or geofence",
+                    "example": "Samsara HQ"
+                },
+                "formattedAddress": {
+                    "type": "string",
+                    "description": "The full address associated with this address/geofence, as it might be recognized by maps.google.com",
+                    "example": "350 Rhode Island St, San Francisco, CA"
+                },
+                "geofence": {
+                    "$ref": "#/definitions/AddressGeofence"
+                }
+            }
+        },
+        "AddressGeofence": {
+            "type": "object",
+            "description": "The geofence that defines this address and its bounds. This can either be a circle, or a polygon - only one key should be provided, depending on the geofence type.",
+            "properties": {
+                "circle": {
+                    "type": "object",
+                    "description": "Information about a circular geofence. This field is only populated if the geofence is a circle.",
+                    "properties": {
```

Comment:
shouldn't we include radius as a property?

Reply:
Yes, this should include `radiusMeters`.

Reply:
:facepalm:

Reply:
I'll add this 😁

Reply:
Added!